frontend_lib/src/component/Avatar/Avatar.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import classnames from 'classnames'
+import { translate } from 'react-i18next'
 
-// require('./Avatar.styl') // see https://github.com/tracim/tracim/issues/1156
+require('./Avatar.styl')
```

Comment:
Why this is now required?

Reply:
I added  the styling for the image (border-radius) inside.
I read the linked issue, it isn't valid anymore as we already use `require()` of stylus files in many frontend_lib components.